changelog.txt

```diff
@@ -1,3 +1,27 @@
+1.11.2 (2021-05-03)
+------------------------------------------------------------------------
+Change: [Win32] Limit hardware accelerated video driver to OpenGL 3.2 or higher (#9077)
+Change: More improvements to the GUI at different scales (#9075, #9102, #9107, #9133, #9174, #9183)
+Fix: Query windows could be partially drawn initially (#9184)
+Fix #9113: Potential crash when removing an airport that exists in an aircraft's orders (#9182)
```

Comment:
You use "potential" a few times .. I think it a weird way to describe a bug and/or fix. Like "we are not sure this could happen, but if it did, it no longer does". Pretty sure that is often not the case ;)

I think we should just remove "potential" everywhere, before every line is invested with that word :D